buildSrc/src/main/java/org/elasticsearch/gradle/internal/test/DistroTestPlugin.java

```diff
@@ -58,8 +58,8 @@
 public class DistroTestPlugin implements Plugin<Project> {
     private static final String SYSTEM_JDK_VERSION = "11.0.2+9";
     private static final String SYSTEM_JDK_VENDOR = "openjdk";
-    private static final String GRADLE_JDK_VERSION = "14+36@076bab302c7b4508975440c56f6cc26a";
-    private static final String GRADLE_JDK_VENDOR = "openjdk";
+    private static final String GRADLE_JDK_VERSION = "15.0.2+7";
```

Comment:
I couldn't get the jdkdownloadplugin to download an openjdk 15 one. Assuming adoptopenjdk is also fine

Reply:
adoptopenjkd should be fine.